cli/user/README.md

```diff
@@ -66,6 +66,7 @@ Initial configuration:
 
 Type "help" for list of commands, or "exit" to exit.
 ```
+The values can be changed at the properties file, or use `config set property=...` to change the value. Make sure to update `controller-uri:<pravega-cluster-name>-pravega-controller.<namespace>:<port>`
```

Comment:
I'm looking to the config directory of Pravega and I don't see any configuration file related to the Pravega User CLI. Inspecting the code, it looks like it does not loads any configuration neither. Apparently, the Pravega User CLI just expects the configuration to be set via `config set`, so please fix in the document any mention to any configuration file in this README.